Tasks/DotNetCoreInstallerV0/externals/install-dotnet.sh

```diff
@@ -654,7 +658,19 @@ install_dotnet() {
     say_verbose "Zip path: $zip_path"
 
     say "Downloading link: $download_link"
-    download "$download_link" $zip_path || download_failed=true
+    download "$download_link" $zip_path || fallback_download_failed=true
```

Comment:
? This should be "download_failed=true"

Reply:
there is a legacy download link, so I added two variables 
I need to swap the names fallback_download_failed and download_failed

Reply:
updated the variables name